src/App.js

```diff
@@ -1,45 +1,103 @@
 import './App.css';
 import { useState, useEffect } from 'react';
 import axios from 'axios';
-import Search from './Search.js';
+import Results from './Results';
 
 function App() {
   const [error, setError] = useState(null);
   const [movies, setMovies] = useState([]);
   const [userInput, setUserInput] = useState('');
-  const [searchQuery, setSearchQuery] = useState('-');
+  const [searchQuery, setSearchQuery] = useState('');
 
-  // we only want our API to call when user submits the search form
+  const [people, setPeople] = useState([]);
+  const [userInput2, setUserInput2] = useState('');
```

Comment:
Simple and effective way of avoiding confusion while naming variables so good job!